workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/WorldCommandRequestSender.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Improbable.Gdk.Core.GameObjectRepresentation;
+using Improbable.Worker;
+using Unity.Entities;
+
+#region Diagnostic control
+
+// ReSharper disable MemberHidesStaticFromOuterClass
```

Comment:
Ooh I hadn't thought of this problem. I actually don't think we want to get rid of this warning is it really is us doing something wrong.

Reply:
I think we can get rid of the partial keyword from Reacquirables and it will do the same thing.

Reply:
At least we shouldn't disable resharper warnings for this. We can get rid of warnings that get passed on to users.

Reply:
The culprits are the methods:

```
public long ReserveEntityIds(uint numberOfEntityIds, uint? timeoutMillis = null, object context = null)
```

`WorldCommands.Requirables.WorldCommandRequestSender#ReserveEntityIds` shadows `WorldCommands.ReserveEntityIds` class

Reply:
Actually are we? This will make the correct code but does it do it in the way we think it does. Will consult the standard.

Reply:
"Nested types may be declared in multiple parts by using the partial modifier. Typically, the containing type is declared using partial as well, and each part of the nested type is declared in a different part of the containing type." - C# language spec

We're all good. Feel free to get rid of the warning.